src/packages.gleam

```diff
@@ -20,6 +21,8 @@ const usage = "Usage:
 "
 
 pub fn main() {
+  let _ = dotenv.load()
```

Comment:
`dotenv.load` will return an error if the `.env` file doesn't exist, so we're ignoring that error (since the environment variables could be provided in another way).